### PR TITLE
fix(relax clean flow cell criteria)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

Relaxes the clean flow cell criteria to either require fastq files OR spring and spring meta data files in housekeeper.

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: tests/meta/clean/test_clean_flow_cells_api.py [hunk 1]
```diff
@@ -172,6 +172,79 @@ def test_can_flow_cell_be_deleted(flow_cell_clean_api_can_be_removed: CleanFlowC
...
+        with mock.patch(
+            "cg.meta.clean.clean_flow_cells.CleanFlowCellAPI.has_spring_meta_data_files_for_samples_in_housekeeper",
+            return_value=False,
+        ):
+            # WHEN checking that the flow cell can be deleted
+            can_be_deleted: bool = (
+                flow_cell_clean_api_can_be_removed.can_flow_cell_directory_be_deleted()
+            )
+
+            # THEN the check whether the flow cell can be deleted returns True
```
Comment (henrikstranneheim) [inline tests/meta/clean/test_clean_flow_cells_api.py:194]: Align THEN with assert for these tests

**Review by henrikstranneheim — APPROVED** [review]
Great 👍

**Comment by ChrOertlin** [comment]
Dry-run test passed

**Comment by ChrOertlin** [comment]
stage: Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/christian.oertlin
done.
Log deploy... done.
cg, version 52.1.1

Prod:
repository is clean
Logging deploy ...
Getting deployer... done.
Getting last commit message and SHA... done.
Getting version of deploy scripts... /home/christian.oertlin
done.
Log deploy... done.
cg, version 52.1.1

**Pull request merged**